lua/telescope/_extensions/zoxide/config.lua

```diff
@@ -12,33 +12,22 @@ local default_config = {
   mappings = {
     default = {
       action = function(selection)
-        vim.cmd.cd(selection.path)
+        vim.cmd("cd " .. selection.path)
       end,
       after_action = function(selection)
         print("Directory changed to " .. selection.path)
-      end
+      end,
     },
     ["<C-s>"] = { action = z_utils.create_basic_command("split") },
     ["<C-v>"] = { action = z_utils.create_basic_command("vsplit") },
     ["<C-e>"] = { action = z_utils.create_basic_command("edit") },
-    ["<C-b>"] = {
-      keepinsert = true,
-      action = function(selection)
-        builtin.file_browser({ cwd = selection.path })
-      end
-    },
     ["<C-f>"] = {
       keepinsert = true,
       action = function(selection)
         builtin.find_files({ cwd = selection.path })
-      end
-    },
-    ["<C-t>"] = {
```

Comment:
Curious why you removed this default mapping as well? It is also not described in the comment or PR message of why.

Reply:
Updated it myself :)